mlflow/store/tracking/rest_store.py

```diff
@@ -339,7 +343,22 @@ def get_trace_info(self, request_id):
         req_body = message_to_json(GetTraceInfo(request_id=request_id))
         endpoint = get_trace_info_endpoint(request_id)
         response_proto = self._call_endpoint(GetTraceInfo, req_body, endpoint=endpoint)
-        return TraceInfo.from_proto(response_proto.trace_info)
+        assessments = None
+        if MLFLOW_TRACKING_URI.get() == "databricks":
```

Comment:
```suggestion
        if is_databricks_uri(mlflow.get_tracking_uri()):
```

1. We should to use [is_databricks_uri](https://github.com/mlflow/mlflow/blob/9cf17478518f632004ae062e87224fea0f704b45/mlflow/utils/uri.py#L83) to include tracking URI with profile/secret info.
2. We should use `mlflow.get_tracking_uri` because it might not be specified via environment variable.